### PR TITLE
Add sdl-generic-hmi.bb (generic_hmi)

### PR DESCRIPTION
**Pull request by gunnarx**
We might need to propose some more changes but let's push this PR to begin with.

### CODE REVIEW

File: recipes-automotive/sdl-generic-hmi/sdl-generic-hmi_git.bb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+"
+
+FILES_${PN}_append = " \
+    ${datadir}/smartdevicelink/generic_hmi/ \
+"
+
+do_install() {
+    install -d ${D}${datadir}/smartdevicelink/generic_hmi/build
+    install -m 0644 ${S}/index.html ${D}${datadir}/smartdevicelink/generic_hmi
+    install -m 0644 ${S}/build/bundle.js ${D}${datadir}/smartdevicelink/generic_hmi/build
```
Comment (phongt) [inline recipes-automotive/sdl-generic-hmi/sdl-generic-hmi_git.bb:29]: @gunnarx  Thanks for the PR. I tested on PC and use Inspect tool on google chrome browser and check the console. It reports some fonts are missing.

> cc56dd09ba0c52a6824aa936024ac810.woff2 Failed to load resource: net::ERR_FILE_NOT_FOUND
> 4e8caa33f4fef80561b267d0b8cd2126.woff2 Failed to load resource: net::ERR_FILE_NOT_FOUND
> 9e06a516bafae247b569ad0c32cd3a57.woff2 Failed to load resource: net::ERR_FILE_NOT_FOUND
> d9f44a669b0ce892cee443367d04adec.woff Failed to load resource: net::ERR_FILE_NOT_FOUND
> d612704147ab38b14c13feaa994a2220.woff Failed to load resource: net::ERR_FILE_NOT_FOUND
> 9b81f8c40379b3fe831234c9895f55d7.woff Failed to load resource: net::ERR_FILE_NOT_FOUND

Please install the fonts folder to ${D}${datadir}/smartdevicelink/generic_hmi/ 

` install -d ${D}${datadir}/smartdevicelink/generic_hmi/fonts`
`install -m 0644 ${S}/fonts/*  ${D}${datadir}/smartdevicelink/generic_hmi/fonts`

Comment (gunnarx) [reply recipes-automotive/sdl-generic-hmi/sdl-generic-hmi_git.bb:29]: I'm not sure.  When asking the slack channel they said the only thing needed is the html and the javascript in the build dir.  I also had reports that we ran it successfully on the target, but I guess I could test it myself once more.

**Comment by gunnarx** [comment]
Hello @phongt Sorry for the delay.  Hope to come back to you and update the PR soon.

**Comment by gunnarx** [comment]
Obsolete, see #4 instead. Closing.

**Pull request closed without merging**